packages/grid/helm/syft/templates/backend/backend-statefulset.yaml

```diff
@@ -152,6 +157,53 @@ spec:
             timeoutSeconds: 5
             failureThreshold: 3
           readinessProbe: null
+        {{- if .Values.attestation.enabled }}
+        - name: enclave-attestation
+          image: {{ .Values.global.registry }}/openmined/grid-enclave-attestation:{{ .Values.global.version }}
+          imagePullPolicy: Always
+          resources: {{ include "common.resources.set" (dict "resources" .Values.attestation.resources "preset" .Values.attestation.resourcesPreset) | nindent 12 }}
+          env:
+            - name: UVICORN_LOG_LEVEL
+              value: {{ .Values.attestation.uvicornLogLevel }}
+            - name: APP_LOG_LEVEL
+              value: {{ .Values.attestation.appLogLevel }}
+            {{- if .Values.attestation.env }}
+            {{- toYaml .Values.attestation.env | nindent 12 }}
+            {{- end }}
+          volumeMounts:
+            - name: kernel-security
+              mountPath: /sys/kernel/security
+            - name: tpm-device
+              mountPath: /dev/tpmrm0
+          securityContext:
+            privileged: true
+          ports:
+            - name: attestation-api
+              containerPort: 4455
+          startupProbe:
+            httpGet:
+              path: /?probe=startupProbe
+              port: attestation-api
+            failureThreshold: 30
+            periodSeconds: 10
+          livenessProbe:
+            httpGet:
+              path: /?probe=livenessProbe
+              port: attestation-api
+            periodSeconds: 15
+            timeoutSeconds: 5
+            failureThreshold: 3
+          readinessProbe: null
+        {{ end }}
+      {{- if .Values.attestation.enabled }}
+      volumes:
+        - name: kernel-security
+          hostPath:
+            path: /sys/kernel/security
+        - name: tpm-device
+          hostPath:
+            path: /dev/tpmrm0
+      {{- end }}
```

Comment:
Any idea what would happen/what error will be thrown if we run this on a node that does not have these files?